frontend/src/metabase/parameters/utils/dashboards.ts

```diff
@@ -0,0 +1,273 @@
+import _ from "underscore";
```

Comment:
Besides `getDashboardUiParameters`, `getMappings`, and `hasMatchingParameters`, every other function in this file has not changed.